### ISSUE TITLE
How to reproduce the results of mlperf v4.0 llama2 70b sparse?

### ISSUE THREAD

**Issue by DehuaTang** [NVIDIA/TensorRT-Model-Optimizer#44]
Outstanding work! Thanks for the effort you guys put in!
Is it really possible to achieve 99.9% accuracy and no fine-tuning for the llama-70b-chat in mlperf task with 2:4 sparse? I have reproduced and tested it using MTO and found that it only achieves 98% accuracy in fp16。
Are you able to give me some suggestions to reproduce this work? Like which hyperparameters need to be adjusted? Is it necessary to use fp8 fine-tuning to achieve 99.9% accuracy?

**Comment by kaix-nv** [comment]
Thank you for your interest. It's feasible to achieve 99.9 ROUGE scores without fine-tuning for the 2:4 sparsified LLaMA-70B-Chat model. We utilized the ModelOpt sparsity package to accomplish this. The key difference may lie in the calibration dataset. We randomly selected a subset from the Open-Orca dataset, ensuring that test samples from MLPERF were excluded.

**Comment by DehuaTang** [comment]
That's amazing!  All the sparse papers with 2:4 sparse don't turn out to be 99.9 accurate. 
Do you have any plans to give the full code? It would help to reintroduce the industry to the impact of sparse on llm

**Comment by cjluo-nv** [comment]
@DehuaTang our code now are fully open sourced. Feel free to reopen if you cannot get the details you need in our code base.
